components/UI/Map/LeafletMap.tsx

```diff
@@ -267,6 +268,7 @@ function LeafletMap(props: ILeafletMap) {
         <LayerControl
           points={points}
           data={data}
+          food={props.food}
           ahbap={props.ahbap}
           hospital={props.hospital}
         />
```

Comment:
if the props are one-on-one, we can also just deconstruct the props
`{...props}`